man-roxygen/DataSource.R

```diff
@@ -0,0 +1 @@
+#' @param dataSource           A list object that is the output of \code{createDatabaseDataSource} function.
```

Comment:
Need to be a bit more verbose here. This would be the place to mention that the data extraction functions can work with a database of a set of files. It would also be good to mention the createFileDataSource function as an alternative to createDatabaseDataSource. 

The notion of dataSources will be tricky for many users, so you'll need to help them a little bit.

Reply:
done